### PR TITLE
Add minutes for 2020-10-29

### PR DESCRIPTION
**Pull request by Adimote**

### CODE REVIEW

File: SR2021/2020-10-29.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+## Game
+- Alistair should have formed ideas into a rulebook (have bludgeoned him in slack)
+## Handling external comms
+- Jake to own team communication
+- James to own public coms
+  - [AI James Seden Smith] email trustees for social media access
+- Andy to help with social media
+
+# Actions
+inlined
```
Comment (PeterJCLaw) [inline SR2021/2020-10-29.md:43]: Suggest duplicating them down here so we can have issues out of them.

File: SR2021/2020-10-29.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
+# Competition Team Meeting 2020-10-15
```
Comment (RealOrangeOne) [inline SR2021/2020-10-29.md:1]:
```suggestion
# Competition Team Meeting 2020-10-29
```

File: SR2021/2020-10-29.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+## Attendees
+- Jenny Fletcher
+- Ash Holland
+- Andy Barrett-Sprot
+- James Seden-Smith
+- Jake Howard
+- Peter Law
+
+## Topics
+### Team signup
```
Comment (PeterJCLaw) [inline SR2021/2020-10-29.md:12]: Empty line between headings please
```suggestion

### Team signup
```

File: SR2021/2020-10-29.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+- We’ve not sent out a form yet, but it will be done this week.
+- [AI Jake Howard] send the email to teams to notify them of the event occurring
+- Then we can add a sign-up form and send it out in an email (in a form that can be forwarded to other teams).
+- [AI James Seden Smith] update sign-up form for 2021, ask for demographic information (rough numbers).
+### Number of teams per organisation?
```
Comment (PeterJCLaw) [inline SR2021/2020-10-29.md:17]: Empty line between headings please:
```suggestion

### Number of teams per organisation?
```

File: SR2021/2020-10-29.md [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+- Ask them how many they want and discuss with them if it’s more than 3 (to avoid a final league of people in the same organisation)
+- Recommend team size of 4 (maybe 3?)
+## When can we close entries?
```
Comment (PeterJCLaw) [inline SR2021/2020-10-29.md:20]: Empty line between headings & keep them at a suitable level please:
```suggestion

### When can we close entries?
```

File: SR2021/2020-10-29.md [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+- We could in theory keep them open until the first deadline, but lets keep them officially open until a couple days before kickstart.
+- Close on 19th (kickstart is on 21st)
+- Give out team accounts on demand of sign-ups.
+- [AI Jake Howard] draft the email to send to team leaders when their team is confirmed
+## Recurring tasks:
```
Comment (PeterJCLaw) [inline SR2021/2020-10-29.md:25]: Empty line between headings & keep them at a suitable level please:
```suggestion

### Recurring tasks:
```

File: SR2021/2020-10-29.md [hunk 7]
```diff
@@ -0,0 +1,52 @@
...
+- It’s a good idea to create tasks based on the recurring tasks repo
+- [AI Peter Law ]: create all the tasks based on the recurring tasks repository (removing things unrelated to a virtual competition.
+## Simulator:
```
Comment (PeterJCLaw) [inline SR2021/2020-10-29.md:28]: Empty line between headings & keep them at a suitable level please:
```suggestion

### Simulator:
```

File: SR2021/2020-10-29.md [hunk 8]
```diff
@@ -0,0 +1,52 @@
...
+- [AI Andy] Try the explosion mitigations (listed in slack by Andy B-S)
+- [AI Jenny]  Check the segfault issue was fixed.
+## Getting new volunteers in:
```
Comment (PeterJCLaw) [inline SR2021/2020-10-29.md:31]: Empty line between headings & keep them at a suitable level please:
```suggestion

### Getting new volunteers in:
```

File: SR2021/2020-10-29.md [hunk 9]
```diff
@@ -0,0 +1,52 @@
...
+- [AI Jake Howard] Send an email to the volunteer mailing list telling them what we’re doing and how we need help
+- We have a lot of tasks to be done, and so getting new volunteers in would be great.
+## Game
```
Comment (PeterJCLaw) [inline SR2021/2020-10-29.md:34]: Empty line between headings & keep them at a suitable level please:
```suggestion

### Game
```

File: SR2021/2020-10-29.md [hunk 10]
```diff
@@ -0,0 +1,52 @@
...
+- Alistair should have formed ideas into a rulebook (have bludgeoned him in slack)
+## Handling external comms
```
Comment (PeterJCLaw) [inline SR2021/2020-10-29.md:36]: Empty line between headings & keep them at a suitable level please:
```suggestion

### Handling external comms
```

File: SR2021/2020-10-29.md [hunk 11]
```diff
@@ -0,0 +1,52 @@
...
+
+# Actions
+- [AI Jake Howard] send the email to teams to notify them of the event occurring
+- [AI James Seden Smith] update sign-up form for 2021, ask for demographic information (rough numbers).
+- [AI Jake Howard] draft the email to send to team leaders when their team is confirmed
+- [AI Peter Law]: create all the tasks based on the recurring tasks repository (removing things unrelated to a virtual competition.
+- [AI Andy] Try the explosion mitigations (listed in slack by Andy B-S)
+- [AI Jenny]  Check the segfault issue was fixed.
+- [AI Jake Howard] Send an email to the volunteer mailing list telling them what we’re doing and how we need help
+- [AI James Seden Smith] email trustees for social media access
```
Comment (PeterJCLaw) [inline SR2021/2020-10-29.md:50]: Suggest reformatting so these can be picked up by the actions tool:
```suggestion
## Action Points
- Jake Howard: send the email to teams to notify them of the event occurring
- James Seden Smith: update sign-up form for 2021, ask for demographic information (rough numbers).
- Jake Howard: draft the email to send to team leaders when their team is confirmed
- Peter Law: create all the tasks based on the recurring tasks repository (removing things unrelated to a virtual competition) (done).
- Andy: Try the explosion mitigations (listed in slack by Andy B-S)
- Jenny:  Check the segfault issue was fixed.
- Jake Howard: Send an email to the volunteer mailing list telling them what we’re doing and how we need help
- James Seden Smith: email trustees for social media access
```
Though you might also need to update the `.name-map.json` for some of these names.

File: SR2021/2020-10-29.md [hunk 12]
```diff
@@ -0,0 +1,52 @@
...
+
+
```
Comment (PeterJCLaw) [inline SR2021/2020-10-29.md:52]: Can we lose this trailing whitespace?
```suggestion
```

**Review by PeterJCLaw — COMMENTED** [review]
Thanks! Content looks mostly correct, though it would be great to have this formatted slightly more consistently.

**Review by sedders123 — APPROVED** [review]

**Pull request merged**